streampipes-extensions/docker-compose.yml

```diff
@@ -48,26 +48,26 @@ services:
     networks:
       spnet:
 
-  pipeline-elements-all-jvm:
+  extensions-all-jvm:
     build:
-      context: ./streampipes-pipeline-elements-all-jvm
+      context: ./streampipes-extensions-all-jvm
       dockerfile: Dockerfile
       args:
         BASE_IMAGE: adoptopenjdk/openjdk8-openj9:alpine
-    image: incubator-streampipes_pipeline-elements-all-jvm:release-validation
+    image: incubator-streampipes_extensions-all-jvm:release-validation
```

Comment:
Can we get rid of the `incubator` part here as well?

Reply:
In my opinion, yes.